external-applications/src/test/java/io/quarkus/ts/external/applications/TodoDemoIT.java

```diff
@@ -1,24 +1,21 @@
 package io.quarkus.ts.external.applications;
 
 import org.apache.http.HttpStatus;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 import io.quarkus.test.bootstrap.RestService;
 import io.quarkus.test.scenarios.QuarkusScenario;
 import io.quarkus.test.scenarios.annotations.DisabledOnNative;
 import io.quarkus.test.services.GitRepositoryQuarkusApplication;
 
-// TODO: enable when Quarkus TODO application migrates to Quarkus 3
-@Disabled("Disabled until TODO application migrates to Quarkus 3")
 @DisabledOnNative(reason = "This scenario is using uber-jar, so it's incompatible with Native")
 @QuarkusScenario
 public class TodoDemoIT {
     private static final String TODO_REPO = "https://github.com/quarkusio/todo-demo-app.git";
     private static final String VERSIONS = "-Dquarkus.platform.group-id=${QUARKUS_PLATFORM_GROUP-ID} -Dquarkus.platform.version=${QUARKUS_PLATFORM_VERSION} ";
-    private static final String DEFAULT_OPTIONS = "-DskipTests=true " + VERSIONS;
+    private static final String DEFAULT_OPTIONS = " -Dquarkus.hibernate-orm.enabled=false -DskipTests=true " + VERSIONS;
```

Comment:
I do not like that we disable hibernate, since that means, that there is a difference between application, what we test and the one, which quarkus users will run. Does running containers create too much of a load during this test?

Reply:
No, but we literally don't use it (tests are green). I'll enable it then.

Reply:
And it also means I'll have to disable it on Windows when Docker not available.

Reply:
(AKA GH CI)